### PR TITLE
Ryan L. | adding Simpsons Movie quote

### PR DESCRIPTION
**Pull request by rlynn523**
#3

### CODE REVIEW

**Comment by NikhilNamal17** [comment]
@rlynn523 your branch has merge conflicts kindly resolve and pr again.

**Comment by rlynn523** [comment]
Conflict resolved

File: data/data.json [hunk 1]
```diff
@@ -1900,10 +1900,15 @@
       "type": "movie",
       "year": 2019
    },
-   {
       "quote": "I just hope my death makes more cents than my life.",
```
Comment (NikhilNamal17) [inline data/data.json]: missing }

Comment (rlynn523) [reply data/data.json]: fixed

**Pull request merged**